docs/cheqd-cli/cheqd-cli-identity-tutorial.md

```diff
@@ -0,0 +1,413 @@
+      "controller": "did:cheqd:<namespace>:<unique-id>",
+      "public_key_multibase": "<verification-public-key-multibase>"
+    }
+  ],
+  "authentication": [
+    "did:cheqd:<namespace>:<unique-id>#<auth-key-alias>"
+  ]
+}
+```
+
+Within this template we will be required to enter a number of fields
+
+  
+
+Where:
+
+*   `<namespace>` - for now it can `testnet` or `mainnet` . For this flow we use `testnet`
+*   `<unique-id>` - identifier, created on the [step](#3-create-unique-id-for-our-did)
+*   `<key-alias>` - a key alias for the verification method identifier
+*   `<verification-public-key-multibase>` - result of this [step](#2-get-multibase58-string)
+*   `<auth-key-alias>` - alias of authentication key.
+
+In our example:
+
+*   `did:cheqd:<namespace>:<unique-id>` - would be `did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe`
+*   `did:cheqd:<namespace>:<unique-id>#<key-alias>` - `did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe#key1`
+*   `<verification-public-key-multibase>` - key from this [step](#2-get-multibase58-string). As result `z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe3Nmod35uua9TE`
+*   `did:cheqd:<namespace>:<unique-id>#<auth-key-alias>` - let it be:
+
+`did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe#key1`
+
+  
+
+#### After these preparations, the base DIDDoc will look like:
+
+  
+
+```
+{
+  "id": "did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe",
+  "verification_method": [
+    {
+      "id": "did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe#key1",
+      "type": "Ed25519VerificationKey2020",
+      "controller": "did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe",
+      "public_key_multibase": "z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe3Nmod35uua9TE"
+    }
+  ],
+  "authentication": [
+    "did:cheqd:testnet:z4Q41kvWsd1JAuPFBff8Dti7P6fLbPZe#key1"
+  ]
+}
+```
+
+  
+
+We recommend you store this DIDDoc in a separate file, like `json.txt` and inject it while running the command for sending.
+
+  
+
+### 5\. Send DIDDoc to the pool
+
+  
+
+Now that we have our DIDDoc prepared we can send it to the pool.
+
+  
+
+We can use the following command to send the DIDDoc:
+
+  
+
+```
+$ cheqd-noded tx cheqd create-did "$(cat json.txt)" \
+"did:cheqd:testnet:zJ5EDiiiKWDyo79n#key1" \
+--ver-key "FxaJOy4HFoC2Enu1SizKtU0L+hmBRBAEpC+B4TopfQoyetOF5T68Ks3db5Yy9ykFdgEboPUes3m6wvXNLpbv+Q==" \
+  --from  --node https://rpc.testnet.cheqd.network:443 --chain-id cheqd-testnet-4 --fees 5000000ncheq
+```
+
+  
+
+Where:
+
+*   `"did:cheqd:testnet:zJ5EDiiiKWDyo79n#key1"` is the `id` of `verification_method` section
+*   `--ver-key` - is from `keys.txt` file from the [step](#1-generate-verification-key), `priv_key_base_64` field.
+*   Instead of `--fees` option, the `--gas-prices "25ncheq"` can be used also.
+*   `--from` - should be an alias of your cosmos keys.
+
+  
+
+After you execute the command you will receive `"code": 0"`if the DID was successfully written to the ledger. We can do a full query to check this as well.
```

Comment:
What happens if it fails? Is there a consistent error message / code if that happens?

Reply:
Added small description how to process error codes.